cfme/tests/storage/test_manager.py

```diff
@@ -97,10 +99,42 @@ def test_storage_manager_delete(manager, provider_cleanup):
         assignee: mmojzis
         initialEstimate: 1/4h
         casecomponent: Cloud
+        caseimportance: medium
     """
     manager.delete()
     view = manager.create_view(StorageManagerDetailsView)
     view.flash.assert_success_message(
         "Delete initiated for 1 Storage Manager from the CFME Database"
     )
     assert not manager.exists
+
+
+def test_storage_manager_navigation_from_cloudprovider(manager, provider):
+    """ Test whether Storage Manager is accessible from Cloud Provider
+
+    prerequisites:
+        * Storage provider
+
+    Steps:
+        * Go to Cloud Provider summary
+        * Check whether Cloud Provider has any Storage Managers
+        * Click on Storage managers
+        * Select Storage Manager from list
+        * Check whether Storage Manager's Summary is displayed correctly
+
+    Polarion:
+        assignee: mmojzis
+        initialEstimate: 1/4h
+        casecomponent: Cloud
+        caseimportance: high
+    """
+    view = navigate_to(provider, 'Details')
+    manager_count = int(view.entities.summary("Relationships").get_text_of("Storage Managers"))
+    assert manager_count > 0
+    view.entities.summary("Relationships").click_at("Storage Managers")
+    storage_view = view.browser.create_view(ProviderStorageManagerAllView)
```

Comment:
Question:  Why not make a `storage_manager` object and then navigate via that object? Will the navigation be different in that case? (i.e. won't click the `storage_manager` from the provider summary page)

Reply:
He is testing that you can navigate to the Storage Managers page via the Cloud Providers by click the "Relationships" table. This is going to be a different navigate than just navigating to the storage_manager object directly.

Reply:
Yes, that's what I do there.